scrapy/dispatch/weakref_backports.py

```diff
@@ -0,0 +1,67 @@
+"""
+weakref_backports is a partial backport of the weakref module for python
+versions below 3.4.
```

Comment:
Aha, makes sense. Maybe we should raise an issue at https://github.com/pjdelport/backports.weakref, or even contribute a PR with WeakMethod changes. Less code we have in Scrapy the better :) What do you think?